### ISSUE TITLE
Move or remove manualtests in testsuites

### ISSUE THREAD

**Issue by heplesser** [nest/nest-simulator#302]
The main and the topology testsuites currently contain tests that need to be run manually. In practice, this means that they are hardly ever run and quickly become outdated, see comment in #221.

Those tests should preferably be converted to proper unit or regression tests, or, where that would be impossible, removed. Removal appears preferable to keeping dead code around.

**Comment by jakobj** [comment]
one of the outdated manualtests seems to be "testsuite/manualtests/test_stdp_dopa.py", which does not seem to produce the desired behaviour:

```bash
expected weight at T=1000 ms: 28.6125 pA
weight at last event: 0.0 pA
```

**Comment by heplesser** [comment]
Manual topology tests are now included in the testsuite per #705. Other manual test are addressed in the context of #348. I therefore close this issue.
